xds/internal/xdsclient/transport/transport.go

```diff
@@ -268,7 +271,11 @@ func (t *Transport) sendAggregatedDiscoveryServiceRequest(stream adsStream, reso
 	if err := stream.Send(req); err != nil {
 		return fmt.Errorf("sending ADS request %s failed: %v", pretty.ToJSON(req), err)
 	}
-	t.logger.Debugf("ADS request sent: %v", pretty.ToJSON(req))
+	if t.logger.V(perRPCVerbosityLevel) {
+		t.logger.Infof("ADS request sent: %v", pretty.ToJSON(req))
```

Comment:
Should the log msgs be swapped around? We want to see the whole resp pretty printed in debug, while only the gist in Infof()? In my experience this pretty printed JSON is pretty cluttery :P

Let me know if im missing something

Reply:
We actually don't have a `DEBUG` log level. See: https://github.com/grpc/grpc-go/blob/master/Documentation/log_levels.md

The `Debugf()` method on the logger is a convenience or shorthand for `INFO` + `verbosity=2`. Given that, what we are doing here is that if the verbosity is high enough for verbose per-RPC messages, we print that as `Info()`, because doing a `Debug()` only ends up checking the verbosity level against `2` and print at `Info()`. And if the verbosity is not high enough, we print a terse message at `Info+2`.

Does this make sense?

Reply:
Yes it does. Thanks for explaining.